ci.template

```diff
@@ -0,0 +1,47 @@
+
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "mapbox-events-ios",
+    "Resources": {
+        "CIUser": {
+            "Type": "AWS::IAM::User",
+            "Properties": {
+                "Policies": [
+                    {
+                        "PolicyName": "publish-metrics",
+                        "PolicyDocument": {
+                            "Statement": {
+                                "Effect": "Allow",
+                                "Action": "*",
```

Comment:
Please scope the `Action` down if you can, if you're only writing to S3 try `s3:PutObject*`.